src/app/hearing-part/components/listing-create/listing-create.component.ts

```diff
@@ -106,4 +118,8 @@ export class ListingCreateComponent {
             }, this.targetDatesValidator)
         })
     }
+
+    private asArray(data) {
```

Comment:
I think that it was raised multiple times and no one took care of that, but this is probably something that could be extracted to a helper or anything like that.